model-optimizer/extensions/middle/MakeKaldiConstReshapable.py

```diff
@@ -97,9 +97,9 @@ def find_and_replace_pattern(self, graph: Graph):
 
         # check that all Parameters have the same batch
         for p in params:
-            assert(p.shape[0] == batch,
-                   "Parameter {} have batch different from the {}".format(p.soft_get('name', p.id),
-                                                                          params[0].soft_get('name', params[0].id)))
+            assert p.shape[0] == batch, \
+                   "Parameter {} has batch different from the {}".format(p.soft_get('name', p.id),
+                                                                          params[0].soft_get('name', params[0].id))
```

Comment:
@sadolini , could you perform an experiment why pylint didn't find the issue with the original code? I expect that it should treat such a problems as errors. May be some pylint command line parameters should be specified to avoid such an issues in the future?

Reply:
ok, I will check

Reply:
pylint detect it as warning, I don't see any way to convert warning to error, but possibly we can to fail in pre-commit if we have warnings in pylint? Currently we have no warnings.

Reply:
Yes, let's enable fail with warnings. @ababushk , could you change the pre-commit scripts to fail if warnings exist?